mlmicrophysics/models.py

```diff
@@ -30,14 +30,15 @@ class DenseNeuralNetwork(object):
         verbose: Level of detail to provide during training
         model: Keras Model object
     """
-    def __init__(self, hidden_layers=1, hidden_neurons=4, activation="relu",
+    def __init__(self, hidden_layers=1, hidden_neurons=4, activation="relu", leaky_alpha=0.1,
                  output_activation="linear", optimizer="adam", loss="mse", use_noise=False, noise_sd=0.01,
                  lr=0.001, use_dropout=False, dropout_alpha=0.1, batch_size=128, epochs=2,
```

Comment:
It would be more ideal to set the default for epochs to 2 in case someone forgets to specify it in their config file. This would make the epoch-by-epoch hyperparameter optimization run twice as long because it would train each network for 2 epochs per iteration rather than 1.

Reply:
I don't think I'm following this one - to me it looks like the default for epochs is already set to 2 but let me know what I'm missing of if I've overwritten it somewhere.